monkey/tests/unit_tests/infection_monkey/ransomware/test_ransomware_payload.py

```diff
@@ -163,6 +163,24 @@ def test_encryption_skipped_if_configured_false(
     assert hash_file(ransomware_target / TEST_KEYBOARD_TXT) == TEST_KEYBOARD_TXT_CLEARTEXT_SHA256
 
 
+def test_encryption_skipped_if_no_directory(
+    ransomware_payload_config, telemetry_messenger_spy, monkeypatch
+):
+    ransomware_payload_config["encryption"]["enabled"] = True
```

Comment:
This is true by default in `ransomware_payload_config`.

Reply:
I'd still prefer to explicitly state that so that there's no possibility of us passing the test just because the defaults changed.